benchmark/opperf/nd_operations/README.md

```diff
@@ -0,0 +1,107 @@
+<!--- Licensed to the Apache Software Foundation (ASF) under one -->
+<!--- or more contributor license agreements.  See the NOTICE file -->
+<!--- distributed with this work for additional information -->
+<!--- regarding copyright ownership.  The ASF licenses this file -->
+<!--- to you under the Apache License, Version 2.0 (the -->
+<!--- "License"); you may not use this file except in compliance -->
+<!--- with the License.  You may obtain a copy of the License at -->
+
+<!---   http://www.apache.org/licenses/LICENSE-2.0 -->
+
+<!--- Unless required by applicable law or agreed to in writing, -->
+<!--- software distributed under the License is distributed on an -->
+<!--- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY -->
+<!--- KIND, either express or implied.  See the License for the -->
+<!--- specific language governing permissions and limitations -->
+<!--- under the License. -->
+
+# TODO - Operators to add to this Benchmark Utility
+
+Below are list of operators to be added to the benchmarking utility:
+
+**NN Operations**
+
+MXNet NDArray NN Operators
```

Comment:
Since this is now automated, is this doc needed anymore? Docs get stale quickly

Reply:
Agree. Missed to remove this. Updated with autogenerated list of operators not covered from the utility